src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+function findSelectedTab(tabs, selectedTabId) {
+  return tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+}
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = findSelectedTab(tabs, selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({ 'is-active': selectedTab.id === tab.id })}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (selectedTabId !== tab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {findSelectedTab(tabs, selectedTabId)?.content}
```

Comment:
To improve readability, it is worth transferring it to a separate variable and using the finished result.